tests/devices/qutrit_mixed/test_qutrit_mixed_measure.py

```diff
@@ -0,0 +1,507 @@
+            )
+
+        expected = []
+        for i in range(BATCH_SIZE):
+            expval_sum = 0.0
+            for summand in observable:
+                expval_sum += get_expval(summand, two_qutrit_batched_state[i])
+            expected.append(expval_sum)
+
+        assert qml.math.allclose(res, expected)
+
+    def test_expval_hamiltonian_measurement(self, two_qutrit_batched_state):
+        """Test that expval Hamiltonian measurements work on broadcasted state"""
+        observables = [qml.GellMann(1, 1), qml.GellMann(0, 6)]
+        coeffs = [2, 0.4]
+        observable = qml.Hamiltonian(coeffs, observables)
+        res = measure(qml.expval(observable), two_qutrit_batched_state, is_state_batched=True)
+
+        expanded_mat = np.zeros((9, 9), dtype=complex)
+        for coeff, summand in zip(coeffs, observables):
+            mat = summand.matrix()
+            expanded_mat += coeff * (
+                np.kron(np.eye(3), mat) if summand.wires[0] == 1 else np.kron(mat, np.eye(3))
+            )
+
+        expected = []
+        for i in range(BATCH_SIZE):
+            expval_sum = 0.0
+            for coeff, summand in zip(coeffs, observables):
+                expval_sum += coeff * get_expval(summand, two_qutrit_batched_state[i])
+            expected.append(expval_sum)
+
+        assert qml.math.allclose(res, expected)
+
+    @pytest.mark.parametrize(
+        "observable",
+        [
+            qml.GellMann(0, 1),
+            qml.GellMann(1, 6),
+            (qml.GellMann(0, 6) @ qml.GellMann(1, 2)),
+        ],
+    )
+    def test_variance_measurement(self, observable, two_qutrit_batched_state):
+        """Test that variance measurements work on broadcasted state."""
+        res = measure(qml.var(observable), two_qutrit_batched_state, is_state_batched=True)
+
+        obs_squared = qml.prod(observable, observable)
+
+        expected = []
+        for state in two_qutrit_batched_state:
+            expval_obs = get_expval(observable, state)
+            expval_of_squared_obs = get_expval(obs_squared, state)
+            expected.append(expval_of_squared_obs - expval_obs**2)
+        assert np.allclose(res, expected)
+
+
+obs_list = [
+    qml.GellMann(0, 3) @ qml.GellMann(1, 1),
+    qml.GellMann(1, 6),
+    qml.GellMann(1, 8),
+    qml.GellMann(1, 7),
+]
+measurement_processes = [
+    qml.expval(qml.GellMann(0, 3)),
+    qml.expval(
+        qml.Hamiltonian(
+            [1.0, 2.0, 3.0, 4.0],
+            obs_list,
+        )
+    ),
+    qml.expval(
+        qml.dot(
+            [1.0, 2.0, 3.0, 4.0],
+            obs_list,
+        )
+    ),
+    qml.var(qml.GellMann(0, 3)),
+    # qml.var(                                  causing error do to bug #5032
+    #     qml.dot(
+    #         [1.0, 2.0, 3.0, 4.0],
+    #         obs_list,
+    #     )
+    # ),
+]
+probs_processes = [qml.probs(wires=0), qml.probs(op=qml.GellMann(0, 8)), qml.probs(wires=[0, 1])]
+
+
+class TestNaNMeasurements:
```

Comment:
I added them because they weren't to difficult to add and I saw it as an eassy place to add some interface testing, I can remove them if you like or keep them as is, whatever you see as best.